mypy/nodes.py

```diff
@@ -457,6 +476,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             arg.set_line(self.line)
         return self
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
```

Comment:
perhaps this should be removed for now, until I figure out how (in FLUP?) to set columns for args